android/build.gradle.kts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) react-native-community
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import com.android.build.gradle.internal.tasks.factory.dependsOn
+import groovy.json.JsonSlurper
+import org.apache.tools.ant.taskdefs.condition.Os
+import java.util.Properties
+
+val reactNativeDir = findNodePackageDir("react-native")
+val reactNativeManifest = file("${reactNativeDir}/package.json")
+val reactNativeManifestAsJson = JsonSlurper().parseText(reactNativeManifest.readText()) as Map<*, *>
+val reactNativeVersion = reactNativeManifestAsJson["version"] as String
+val (major, minor, patch) = reactNativeVersion.split(".")
+val rnMinorVersion = minor.toInt()
+val rnPatchVersion = patch.toInt()
+val prefabHeadersDir = file("${layout.buildDirectory.get()}/prefab-headers")
```

Comment:
Q: Where is this coming from?

Reply:
you mean `layout.buildDirectory.get()`? having a warning from android studio.
![Screenshot 2025-02-15 at 12 00 13 AM](https://github.com/user-attachments/assets/0b0812aa-4868-4257-a4ef-8b51eff0ae5e)

originally the prefabHeadersDir basically referenced from your code on ReactAndroid